Database/Document.ts

```diff
@@ -23,4 +23,31 @@ export namespace Document {
 		const { id, created, changed, purged: purged, ...remainder } = document
 		return [{ id, created, changed, purged }, remainder as T]
 	}
+	export function update<T extends Record<string, any> = Document>(original: T, appendee: Partial<T>): T | undefined {
+		return JSON.parse(JSON.stringify({ ...original, ...appendee }))
```

Comment:
Why stringify and parse the object?

Reply:
There was a bug and this was my way of making a deep copy :)

Reply:
Nice :D